assets/js/timetree.js

```diff
@@ -512,11 +512,26 @@ function TreeGraph({ nodes, links, centuries }) {
 
   const panel = document.querySelector("#panel");
   d3.select("aside .close").on("click", function () {
+    // Close panel and deselect
+    close_panel(panel);
+  });
+
+  // Close panel function
+  function close_panel(panel) {
```

Comment:
makes sense to have this as a function! for more javascripty style we might want to call it `closePanel`, and if we end up with more panel behaviors we might want to move it (maybe panel + tag behavior), but this seems fine to me for now

Reply:
ooh good call, I hardly ever use camelCase but for javascript (because of java?) I suppose that's the normative style